src/main/scala/karazin/scala/users/group/week2/homework/program.scala

```diff
@@ -6,28 +6,47 @@ import model._
 import services._
 
 object program:
-  
-  /*
-   Print all view for all user's posts if they exists
-  */
+
+  //I`m really tried :(
+  //Still not work
+  //Getting view for all user's posts if they exists
+  def getPostsViews(): ErrorOr[List[PostView]] =
```

Comment:
Let's discuss the solution tomorrow.
It's a very common approach in a industrial programming, that's why I want you to learn it.